project/BRANCHES-AND-TAGS.md

```diff
@@ -1,32 +1,39 @@
 Branches and tags
 =================
 
-Note: details of the release process for the Engine are documented in the
-[PATCH-RELEASES.md](https://github.com/moby/moby/blob/master/project/PATCH-RELEASES.md).
+> Note: details of the release process of the project are documented in [PATCH-RELEASES.md](PATCH-RELEASES.md).
 
 # Branches
 
-The docker/docker repository should normally have only three living branches at all time, including
-the regular `master` branch:
+`master` serves as the development branch for future releases of the project.
+All changes should be made to the `master` branch, and changes to release branches should only be made in the form of cherry-picked commits, if possible.
+The sponsoring maintainers of a release branch serve as the primary point of contact, and are available to provide guidance on contributing changes to their respective branches.
 
-## `docs` branch
+Keep in mind that release branches only accept bug and security fixes; new features will generally not be considered for backport to release branches.
 
-The `docs` branch supports documentation updates between product releases. This branch allow us to
-decouple documentation releases from product releases.
+Currently (and previously) maintained release branches are documented in the table below:
 
-## `release` branch
+| Branch Name                       | Sponsoring Maintainer(s)                                | Contribution Status   | Expected End of Maintenance  | Known Distributors                  |
+|-----------------------------------|---------------------------------------------------------|-----------------------|------------------------------|-------------------------------------|
+| master (development branch)       | The Moby Project [MAINTAINERS](../MAINTAINERS)          | N/A                   | -                            | N/A                                 |
+| 27.0                              | The Moby Project [MAINTAINERS](../MAINTAINERS)          | Maintained            | After 28.0                   | [Docker, Inc.](https://docker.com)  |
```

Comment:
```suggestion
| 27.0                              | The Moby Project [MAINTAINERS](../MAINTAINERS)          | Maintained            | After 28.0                   | [Docker, Inc.](https://docker.com)  |
```

I put `The Moby Project` as the sponsoring maintainers for the latest release branch to avoid listing every individual maintainer that happens to do the backports.
Alternative would be to create a team in the Moby org with people that do backports (which would be at least Docker Inc. maintainers?).

@thaJeztah  @cpuguy83  @corhere  WDYT?